go.mod

```diff
@@ -1,20 +1,30 @@
 module github.com/google/zoekt
 
 require (
+	cloud.google.com/go v0.44.0 // indirect
 	github.com/andygrunwald/go-gerrit v0.0.0-20181026193842-43cfd7a94eb4
+	github.com/chzyer/readline v0.0.0-20180603132655-2972be24d48e // indirect
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/gfleury/go-bitbucket-v1 v0.0.0-20181102191809-4910839b609e
+	github.com/gobwas/glob v0.2.3 // indirect
 	github.com/google/go-cmp v0.3.0
 	github.com/google/go-github/v27 v27.0.4
 	github.com/google/slothfs v0.0.0-20170112234537-ecdd255f653d
+	github.com/hanwen/go-fuse v1.0.0 // indirect
 	github.com/hashicorp/go-retryablehttp v0.5.1
+	github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6 // indirect
 	github.com/keegancsmith/rpc v1.1.0
+	github.com/kisielk/gotool v1.0.0 // indirect
 	github.com/kylelemons/godebug v0.0.0-20170820004349-d65d576e9348
+	github.com/libgit2/git2go v0.0.0-20190618093925-b2e2b2f71bb4 // indirect
 	github.com/mitchellh/mapstructure v1.1.2 // indirect
+	github.com/nbutton23/zxcvbn-go v0.0.0-20180912185939-ae427f1e4c1d // indirect
+	github.com/stretchr/objx v0.2.0 // indirect
 	github.com/xanzy/go-gitlab v0.13.0
 	go.uber.org/automaxprocs v1.2.0
-	golang.org/x/net v0.0.0-20190311183353-d8887717615a
-	golang.org/x/oauth2 v0.0.0-20181120190819-8f65e3013eba
-	golang.org/x/sync v0.0.0-20190227155943-e225da77a7e6
+	golang.org/x/net v0.0.0-20190620200207-3b0461eec859
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/sync v0.0.0-20190423024810-112230192c58
+	golang.org/x/tools v0.0.0-20190809145639-6d4652c779c4 // indirect
 	gopkg.in/src-d/go-git.v4 v4.8.0
```

Comment:
whats going on here? I'm surprised this file changed, you don't change any deps.

Reply:
Hmm, I think this file changed when I ran `go test ~/Code/Sourcegraph/zoekt/...`. I'll remove this from the PR.